codespell_lib/data/dictionary.txt

```diff
@@ -2876,12 +2921,17 @@ hopfully->hopefully
 hoppefully->hopefully
 horphan->orphan
 horrifing->horrifying
+horzontal->horizontal
+horzontally->horizontally
 hosited->hoisted
 hospitible->hospitable
 hould->hold, should,
 hounour->honour
 housr->hours, house,
 hove->hove, have, hover, love,
+hovewer->however
+how log->how long
```

Comment:
this one has legitimate uses

Reply:
@larsoner I was trying to think of some. Care to give an example ?

Reply:
"We have to do this because of how log files work"